action.yml

```diff
@@ -40,6 +40,32 @@ inputs:
   force-new-deployment:
     description: 'Whether to force a new deployment of the service. Valid value is "true". Will default to not force a new deployment.'
     required: false
+  #one off task inputs
+  run-task:
+    description: 'Whether to run the task outside of an ECS service. Task will run before the service is updated if both are provided. Will default to not run.'
```

Comment:
Nit: I think we should mention also if this is a boolean/string here, just like other descriptions. Following a pattern like:
`A boolean indicating whether ... Default value is False ...`